web/themes/new_weather_theme/templates/block/block--weathergov-hourly-forecast.html.twig

```diff
@@ -60,8 +60,13 @@
             {% for period in content.hours %}
               <td style="white-space:pre;">
                 <div class="display-flex flex-row flex-align-center">
-                  <span>{{period.windSpeed}}</span>
-                  <svg role="img" aria-hidden="true" class="width-2 height-2 margin-left-05 padding-2px" data-wind-direction="{{period.windDirection}}">
+                  <span>{{period.windSpeed}} {{ "mph" | t }}</span>
+                  {#
+                    The +180 here is to account for the fact that wind direction
+                    is reported as a FROM direction, but our arrow points in the
+                    TO direction. So we just need to spin it 'round.
+                  #}
+                  <svg role="img" aria-hidden="true" class="width-2 height-2 margin-left-05 padding-2px" style="transform: rotate({{ period.windDirection.angle + 180 }}deg);">
```

Comment:
I'm cool with having it inline like this for readability, but the stylesheet was designed to read from the `data-wind-direction` attribute and apply the correct rotation already ([see here](https://github.com/weather-gov/weather.gov/blob/main/web/themes/new_weather_theme/assets/sass/_uswds-theme-custom-styles.scss#L123-L151))

Reply:
Understood. This code uses the actual angle rather than just the 8 cardinal/ordinal directions. I don't feel strongly about which we should use, though.